build/Microsoft.DotNet.Cli.BundledSdks.props

```diff
@@ -3,7 +3,10 @@
   
   <ItemGroup>
     <!-- CLI cannot use the latest SDK until we move away from SDK PackageRef -->
+    <BundledSdk Include="NuGet.Build.Tasks.Pack" Version="4.0.0-rc2" />
     <BundledSdk Include="Microsoft.Net.Sdk" Version="1.0.0-alpha-20161202-1" />
```

Comment:
Microsoft.Net.Sdk version should be 20161203-1 to match the version of other packages.